packages/@lwc/template-compiler/src/__tests__/fixtures/attributes/scoped-id/expression/not-scoped/actual.html

```diff
@@ -0,0 +1,9 @@
+<template>
+    <div 
+        aria-described-by={scoped}
+        aria-active-descendant={scoped}
+        aria-error-message={scoped}
+        aria-flow-to={scoped}
+        aria-labelled-by={scoped}
```

Comment:
Oh man, it took me a while to realize why these are invalid. Extra hyphens in the attribute names. 🤦 

Why did you decide to add this?

Reply:
I actually intended to add these tests for #4056 but it looks like it snuck in here too.

The goal was to show non-scoped tokens when the aria attributes are assigned expressions as a parallel to the [static attribute fixtures](https://github.com/salesforce/lwc/tree/master/packages/%40lwc/template-compiler/src/__tests__/fixtures/attributes/scoped-id/not-scoped).

Reply:
I would add an HTML comment to both files. It's extremely non-obvious that these should _not_ be scoped since `aria-flow-to` is not a valid aria attribute. (Should be `aria-flowto`.)